usmqe/api/tendrlapi/common.py

```diff
@@ -212,20 +212,24 @@ def get_nodes(self):
         self.check_response(response)
         return response.json()
 
-    def import_cluster(self, cluster_data):
-        """ Import cluster defined by json.
+    def import_cluster(self, sds_type, nodes):
```

Comment:
Thanks, unfortunately there's a problem. Please make the `sds_type` as optional parameter - `sds_type=None`
And if it stays None, don't send it in the request. So we could create a test later where we check if detection works.